### PR TITLE
add build script for convenience

### PR DESCRIPTION
**Pull request by thejerrytan**
added a shell script to build, source env and run roslaunch all in one command. To use this, just run

./build.sh

@pkorivi i think you can add command line flags support and pass it to the line where we call roslaunch in order to control env (simulator or carla) and models choice

### CODE REVIEW

File: ros/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+cd ~/CarND-CapStone/ros
```
Comment (ser94mor) [inline ros/build.sh:1]: I have it in a different directory. Others may also keep it in different directories.

File: ros/build.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+catkin_make
+source devel/setup.bash
+roslaunch launch/styx.launch
```
Comment (ser94mor) [inline ros/build.sh:4]: Here you are running a simulator. In this case, the script should be named differently, like `build_and_run_sim.sh` or `build_and_run_styx.sh`.

**Review by Elgeweily — APPROVED** [review]

**Review by ser94mor — CHANGES_REQUESTED** [review]

File: ros/build.sh (hunk 1 above)
Comment (pkorivi) [reply ros/build.sh:1]: To execute the script we need to navigate to ros folder anyway, so it's better if we remove this line and keep the rest.

File: ros/build.sh (hunk 2 above)
Comment (pkorivi) [reply ros/build.sh:4]: True, we can have two separate scripts then

**Comment by ser94mor** [comment]
@thejerrytan, are you planning to merge this PR? If yes, please, address the comments. If not, please, close this PR.

**Pull request closed without merging**